apps/main/src/dao/hooks/useUserGaugeWeightVotes.ts

```diff
@@ -0,0 +1,11 @@
+import { useUserGaugeWeightVotesQuery } from '@/dao/entities/user-gauge-weight-votes'
+import type { ChainId } from '@/dao/types/dao.types'
+
+export const useUserGaugeWeightVotes = ({ chainId, userAddress }: { chainId: ChainId; userAddress: string }) => {
+  const { data: userGaugeWeightVotes, ...rest } = useUserGaugeWeightVotesQuery({ chainId, userAddress })
```

Comment:
Not a big deal now, but note that using ... in react query can [lead to extra renders](https://tanstack.com/query/latest/docs/eslint/no-rest-destructuring)